### ISSUE TITLE
Add a command line argument to allow me to set max blobs if Im a proposer

### ISSUE THREAD

**Issue by rolfyone** [hyperledger/besu#7683]
Sometimes there's a confluence of circumstances that ends up meaning that a solo staker may be proposing, with a local EL building, and needing to include 6 blobs currently.

Because of a number of factors, this may mean a large volume of data (roughly 80-100MB for 6 blobs - corrected down after twitter discussion) being pushed out of a home internet which may only be ADSL, and this can ultimately lead to missed block proposals.

As a home staker, I would like to be able to dictate the maximum number of blobs for inclusion in a block that I'm building, so that I can have the best chance of publishing the block for inclusion.

Clearly this still needs to apply normal max rules, so if i set --max-blobs=1000, that would be invalid, but I'd like to be able to do something like set --max-blobs=3, or even --max-blobs=1 if i know that I'm unlikely to be able to publish larger volumes of data given my internet circumstances.

There's also no financial incentive for a block producer to produce large amounts of blob data, so the fact that these large objects can ruin my proposals seems very unfair. I'd rather get a block out in time than miss a proposal completely.

**Comment by Deepcryptodive** [comment]
Yes please! I'm a home staker doing local buildingand would use this.

Currently I'm using Starlink as ISP, and bandwith can be an issue sometimes!

**Comment by jflo** [comment]
There is no option to fine-tune the number of blobs to include in a produced block, however the inclusion of blobs can be disabled altogether using the [existing tx-pool-max-prioritized-by-type configuration option](https://besu.hyperledger.org/public-networks/reference/cli/options#tx-pool-max-prioritized-by-type) but the validator would have to be willing to forfeit the blob fees for their proposed block.

**Comment by benaadams** [comment]
Similar request in Nethermind https://github.com/NethermindEth/nethermind/issues/7502

**Comment by benjaminion** [comment]
> There is no option to fine-tune the number of blobs to include in a produced block, however the inclusion of blobs can be disabled altogether using the [existing tx-pool-max-prioritized-by-type configuration option](https://besu.hyperledger.org/public-networks/reference/cli/options#tx-pool-max-prioritized-by-type)

Can you clarify the syntax for this? When I put the following in the config file (as per the example in the docs):

```
tx-pool-max-prioritized-by-type="BLOB=0"
```

I get this error on startup:

```
TomlInvalidTypeException: Value of 'tx-pool-max-prioritized-by-type' is a string while processing argument at or before arg[1] '/etc/besu/besu.toml' in [--config-file, /etc/besu/besu.toml]: org.apache.tuweni.toml.TomlInvalidTypeException: Value of 'tx-pool-max-prioritized-by-type' is a string
```

I have tried many variants of this (adding escaped quotes around "0", changing up the quotation mark types, omitting the quotation marks...), but no success.

**Comment by jflo** [comment]
You've found a documentation error, thank you. The value for the option is an array:

`tx-pool-max-prioritized-by-type=["BLOB=0","FRONTIER=200"]`

is how it is tested (successfully) in our test suite.

**Comment by jflo** [comment]
Good first issue tag applied, because this is doable today, but would be nice to have a more dedicated cli option for it and clear documentation for people who have this use case.
